### PR TITLE
[CTOR-602] refresh Microsoft DHCP SNMP doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md [hunk 1]
```diff
@@ -5,26 +5,64 @@ title: Microsoft DHCP SNMP
...
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                            |
+|:----------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **App-Microsoft-Dhcp-SNMP-custom** host template |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md:39]:
```suggestion
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/applications-protocol-snmp/) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **App-Microsoft-Dhcp-SNMP-custom** |
```

File: pp/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md [hunk 2]
```diff
@@ -33,103 +70,276 @@ It is possible to filter on the address of a subnet using a REGEXP of the form [
 
 To control your Microsoft DHCP software, the SNMP must be configured.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md:71]: Standard SNMP prerequisites

File: pp/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md [hunk 3]
```diff
@@ -33,103 +70,276 @@ It is possible to filter on the address of a subnet using a REGEXP of the form [
...
+	--critical-pending-offers='' \
+	--verbose
 ```
 
-Expected command output is shown below: 
+The expected command output is shown below:
 
 ```bash
-OK: Subnet '192.168.153.0' addresses usage total: 50 used: 35 (70.00%) free: 15 (30.00%), pending offers: 0 | '192.168.153.0#subnet.addresses.usage.count'=35;;;0;50 '192.168.153.0#subnet.addresses.free.count'=15;;;0;50 '192.168.153.0#subnet.addresses.usage.percentage'=70.00%;0:80;0:90;0;100 '192.168.153.0#subnet.pending.offers.count'=0;;;0;
+OK: All subnets are ok | '*subnets*#subnet.addresses.usage.count'=;;;0;total'*subnets*#subnet.addresses.free.count'=;;;0;total'*subnets*#subnet.addresses.usage.percentage'=%;;;0;100'*subnets*#subnet.pending.offers.count'=;;;0;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md:232]: is the example ok?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md [hunk 4]
```diff
@@ -33,100 +71,276 @@ It is possible to filter on the address of a subnet using a REGEXP of the form [
 
 Afin de contrôler votre application Microsoft DHCP, le SNMP doit être configuré.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md:72]: mettre les prérequis standard SNMP

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md [hunk 5]
```diff
@@ -33,100 +71,276 @@ It is possible to filter on the address of a subnet using a REGEXP of the form [
...
+	--warning-pending-offers='' \
+	--critical-pending-offers='' \
+	--verbose
 ```
 
-Exemple de sortie :
+La commande devrait retourner un message de sortie similaire à :
 
+```bash
+OK: All subnets are ok | '*subnets*#subnet.addresses.usage.count'=;;;0;total'*subnets*#subnet.addresses.free.count'=;;;0;total'*subnets*#subnet.addresses.usage.percentage'=%;;;0;100'*subnets*#subnet.pending.offers.count'=;;;0;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-microsoft-dhcp-snmp.md:233]: exemple ok?

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**